src/vue-timepicker.vue

```diff
@@ -1852,7 +1862,7 @@ export default {
 </script>
 
 <template>
-<span class="vue__time-picker time-picker" :style="inputWidthStyle">
```

Comment:
BTW, although this "**time-picker**" class name is from the legacy version (v0.1.x), I'd suggest we keep it here in case some long-term users are still using this specific class as the selector for overriding the CSS styles in their app/webpage.

Reply:
Alright, absolutely. I had no clue about this, I'll revert it.